tests/storages_tests/test_with_server.py

```diff
@@ -115,7 +118,14 @@ def test_loaded_trials(storage_url: str) -> None:
 )
 def test_store_infinite_values(input: float, expected: float, storage_url: str) -> None:
 
-    storage = optuna.storages.RDBStorage(url=storage_url)
+    storage: Union[RDBStorage, RedisStorage]
+    if storage_url.startswith("redis"):
+        # Only RDB can convert max/min value to inf. It was introduced by #3238.
```

Comment:
A nit-picky, but could you write the URL instead of the PR/Issue number?
```suggestion
        # Only RDB can convert max/min value to inf. It was introduced by 
        # https://github.com/optuna/optuna/pull/3238.
```

Reply:
Fixed.

Reply:
Thank you!